src/commands/wizard/lwcGenerationCommand.ts

```diff
@@ -101,107 +116,117 @@ export class LwcGenerationCommand {
                 await access(landingPagePath);
                 const uem = CommonUtils.loadJsonFromFile(landingPagePath);
                 getSObjectsStatus.sobjects = UEMParser.findSObjects(uem);
+                resolve(getSObjectsStatus);
             } catch (err) {
                 console.warn(
                     `File '${landingPageJson}' does not exist at '${staticResourcesPath}'.`
                 );
-                getSObjectsStatus.error = (err as Error).message;
+                reject((err as Error).message);
             }
-
-            resolve(getSObjectsStatus);
         });
     }
 
     static async checkForExistingQuickActions(): Promise<SObjectQuickActionStatus> {
-        return new Promise<SObjectQuickActionStatus>(async (resolve) => {
-            const results: SObjectQuickActionStatus = { sobjects: {} };
+        return new Promise<SObjectQuickActionStatus>(
```

Comment:
It looks like only the formatting changed. Was it by running prettier?

Reply:
This file changed mostly because of prettier, but also because the lines grew in length some because I added the `reject` block to the Promise.